cloud/src/meta-service/meta_service_resource.cpp

```diff
@@ -575,22 +649,20 @@ void MetaServiceImpl::alter_obj_store_info(google::protobuf::RpcController* cont
         }
         break;
     }
-    case AlterObjStoreInfoRequest::ADD_BUILT_IN_VAULT: {
```

Comment:
Do not delete it, it will be used in the future.

Reply:
It seems useless.